.circleci/verbatim-sources/pytorch-build-params.yml

```diff
@@ -18,3 +18,23 @@ pytorch_params: &pytorch_params
     USE_CUDA_DOCKER_RUNTIME: << parameters.use_cuda_docker_runtime >>
   resource_class: << parameters.resource_class >>
 
+pytorch_ios_params: &pytorch_ios_params
```

Comment:
nit: if the parameters are only used from only one location, it's clearer if they're inlined at the single use site (you do not have to do anything about this comment)